web/src/components/profile/ProfileSummaryCard.tsx

```diff
@@ -27,11 +27,13 @@ export default function ProfileSummaryCard({ chosenName, username, roles }: Prof
             <span className={styles.label}>Roles:</span>
             <div className={styles.roles}>
               {roles && roles.length > 0 ? (
-                roles.map((role) => (
-                  <span key={role} className={styles.role}>
-                    {role}
-                  </span>
-                ))
+                roles
+                  .filter((role) => !role.includes("_")) // skip all object roles
```

Comment:
do we also want to hide staff and base? Not massively helpful to a user but is helpful to us 🤷‍♀️ 
<img width="196" height="39" alt="image" src="https://github.com/user-attachments/assets/ad3e556f-716e-450d-93b4-817a8d25a4ff" />